crates/nu-protocol/src/value/mod.rs

```diff
@@ -1987,7 +2009,7 @@ impl Value {
             }
             (Value::Filesize { val: lhs, .. }, Value::Filesize { val: rhs, .. }) => {
                 if *rhs != 0 {
-                    Ok(Value::Int {
+                    Ok(Value::Filesize {
```

Comment:
I think that has to be unitless
```suggestion
                    Ok(Value::Float {
```

Reply:
In my mind, when someone is using units to do math, they want units in the return. I was going though cleaning up some of these scenarios but wanted to make sure people see this the same way.

Reply:
Multiplying in units seems fine for me but having the units in both dividend and divisor should reduce out the unit. (and having a unit just in the divisor is probably funky if we don't want to introduce inverses for all measurement types)

Reply:
I think I went further down the rabbit hole with my last commit. Sorry if I seem obtuse, but do you have some examples of what doesn't work right with these changes? I'm having a hard time understanding why these changes are not correct and I think examples will help me follow what you're saying.

Reply:
I think i found some "4mb / 4mb" and "4ms / 4ms"

Reply:
I think this is resolved now. Let me know if you still have objections.